ios/ElectronCash/electroncash_gui/ios_native/send.py

```diff
@@ -402,7 +412,7 @@ def textFieldShouldReturn_(self, tf : ObjCInstance) -> bool:
         return True
 
     @objc_method
-    def onPayTo_message_amount_(self, address, message, amount) -> None:
+    def onPayTo_message_amount_opReturn_isRaw_(self, address, message, amount, op_return, op_return_is_raw) -> None:
         # address
         if not self.viewIfLoaded:
             self.queuedPayTo = [address, message, amount]
```

Comment:
We should probably save these to the "queued pay-to" mechanism as well (this mechanism is for when you use the QR but the viewcontroller hasn't initialized yet).

```suggestion
            self.queuedPayTo = [address, message, amount, op_return, op_return_is_raw]
```

Reply:
Got it, I missed this "queued pay-to" mechanism. I did saw it but couldn't figure out what was it for. Thanks for the explanation.

Reply:
> Got it, I missed this "queued pay-to" mechanism. I did saw it but couldn't figure out what was it for. Thanks for the explanation.

Yeah no worries. It took me a few times if te-reading the code to remember what it was for. It’s not entirely obvious and between the arcane nature of objc and IB and all the other stuff going on — it’s easy to forget to pay attention to.

Reply:
Fixed. Added the new arguments to the queuedPayTo list